### PR TITLE
user/curtail: new package

### PR DESCRIPTION
**Pull request by tulilirockz**
honestly kinda funny to see that this tool uses shelling out to actually compress stuff. but well, it is quite useful!

### CODE REVIEW

File: user/oxipng/template.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+pkgname = "oxipng"
+pkgver = "9.1.2"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo-auditable"]
```
Comment (EarldridgeJazzedPineda) [inline user/oxipng/template.py:5]:
```suggestion
hostmakedepends = ["cargo-auditable"]
makedepends = ["rust-std"]
```

File: user/curtail/template.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+pkgname = "curtail"
+pkgver = "1.10.0"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = ["desktop-file-utils", "gettext", "meson", "pkgconf"]
+makedepends = ["gtk4-devel"]
+depends = ["jpegoptim", "libwebp-progs", "oxipng", "pngquant", "python-scour"]
+pkgdesc = "GTK Image compressor"
```
Comment (EarldridgeJazzedPineda) [inline user/curtail/template.py:8]:
```suggestion
pkgdesc = "GTK image compressor"
```

File: user/curtail/template.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+pkgname = "curtail"
+pkgver = "1.10.0"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = ["desktop-file-utils", "gettext", "meson", "pkgconf"]
+makedepends = ["gtk4-devel"]
+depends = ["jpegoptim", "libwebp-progs", "oxipng", "pngquant", "python-scour"]
```
Comment (nekopsykose) [inline user/curtail/template.py:7]: this is missing libadwaita and python-gobject

File: user/oxipng/template.py (hunk 1 above)
Comment (tulilirockz) [reply user/oxipng/template.py:5]: good catch! didnt see that earlier

File: user/curtail/template.py (hunk 3 above)
Comment (nekopsykose) [reply user/curtail/template.py]: i don't mean in makedepends, it's a runtime dependency; this is a python application and opens it via gobject. the -devel's don't do anything here :) 

but fixed it for you, thanks, forgot to add oxipng before...

Comment (tulilirockz) [reply user/curtail/template.py]: oh! didnt know that was a runtime one!

**Pull request merged**